src/go/rpk/pkg/cli/topic/BUILD

```diff
@@ -4,6 +4,7 @@ go_library(
     name = "topic",
```

Comment:
This command should have ducktape tests too, since now there is not functional testing where we actually connect to a kafka/redpanda cluster and do the analysis.

Reply:
Added a ducktape test